cmd/main.go

```diff
@@ -269,3 +272,30 @@ func updateHealthMetricsPeriodically(nsxClient *nsx.Client) {
 		}
 	}
 }
+
+func checkLicense(nsxClient *nsx.Client, interval int, licenseTimeoutNoDFW int) {
```

Comment:
No need to add `licenseTimeoutNoDFW` as parameter

Reply:
done